DSMRloggerAPI.ino

```diff
@@ -274,10 +278,10 @@ void setup()
   Serial.print (WiFi.localIP());
   Serial.println("' voor verdere debugging\r\n");
   
-//=============now test if "!PRDconvert" file exists================
+//=============now test if "convertPRD" file exists================
   if (DSMRfileExist("!PRDconvert", false) )
   {
-    convertPRD2RING();  // convert PRD to RING file(s)
+    convertPRD2RING();
```

Comment:
Toegegeven, het commentaar is wat overbodig maar waarom haal je het weg?
Origineel was/is prima